src/base/common.less

```diff
@@ -4,6 +4,30 @@ html,
 body {
   font-size: @BaseFontSize;
 }
+.main-div {
```

Comment:
Are we sure changes here are not impacting other component styles?

Reply:
reverted the common style changes from common css file